### PR TITLE
[8.0]Add import for virtual and downloadable products as services.

### PR DESCRIPTION
**Pull request by jesusVMayor**

### CODE REVIEW

**Comment by atchuthan** [comment]
:+1:

File: magentoerpconnect/product.py [hunk 1]
```diff
@@ -71,11 +71,11 @@ def product_type_get(self):
         return [
             ('simple', 'Simple Product'),
             ('configurable', 'Configurable Product'),
+            ('virtual', 'Virtual Product'),
+            ('downloadable', 'Downloadable Product'),
```
Comment (atchuthan) [inline magentoerpconnect/product.py]: instead of downloadable, you could use "digital"  [Suggestion]

**Comment by guewen** [comment]
:+1:

**Comment by l-arnold** [comment]
+1 
As a short term workaround, I have found that I can build first as Virtual in Magento , then go in and change to Simple in MySQL, then sync to Odoo, then change back to Virtual in Magento. 

Setting a Service in Odoo is a good idea!

**Comment by StefanRijnhart** [comment]
:+1: Merging this as it looks like the CI breakage is unrelated.

**Pull request merged**